### PR TITLE
Drop PhantomJS from Gruntfile, allow Chrome/Firefox from karma.conf.js

### PR DESCRIPTION
**Pull request by benjaminapetersen**
- Our `karma.conf.js` checks for Travis then will opt for Chrome || Firefox
- Our `Gruntfile` has PhantomJS hard-coded in the task

This PR eliminates PhantomJS from the task, so the `karma.conf.js` can choose the browser.  

Note:  I cannot run the tests locally with Phantom:

screen shot 2017-11-03 at 1 55 45 pm

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -62,7 +62,6 @@ module.exports = function (grunt) {
         unit: {
           configFile: 'test/karma.conf.js',
           singleRun: true,
```
Comment (jwforres) [inline Gruntfile.js]: remove the hanging comma

Comment (benjaminapetersen) [reply Gruntfile.js]: done

**Comment by jwforres** [comment]
LGTM just wait for green travis

**Comment by benjaminapetersen** [comment]
Updating `.travis.yml`, Firefox wasn't running correctly.

**Comment by benjaminapetersen** [comment]
screen shot 2017-11-06 at 4 55 58 pm

**Comment by benjaminapetersen** [comment]
Chrome survived. Trying back to FF.

**Comment by benjaminapetersen** [comment]
Wups. should have pushed to get this one in with the 0.0.74 as well.  Annoying the tests don't run locally ☹️

**Comment by benjaminapetersen** [comment]
[test] again because its been a while.
or
/test cuz i don't remember which bot is paying attention to this repo.

**Pull request merged**